src/init.cpp

```diff
@@ -1078,7 +1086,7 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
 
     // Start the lightweight task scheduler thread
     CScheduler::Function serviceLoop = boost::bind(&CScheduler::serviceQueue, &scheduler);
-    threadGroup.create_thread(boost::bind(&TraceThread<CScheduler::Function>, "scheduler", serviceLoop));
+    scheduler_thread = std::thread(std::bind(&TraceThread<CScheduler::Function>, "scheduler", serviceLoop));
```

Comment:
Ah, I didn't realize that you meant to replace the binds with lambdas. I'm not sure that I find this any less confusing, but I don't really mind either way:
```c++
scheduler_thread = std::thread([&scheduler] {
    TraceThread("scheduler", [&scheduler] {
        scheduler.serviceQueue();
    });
});
```